automatic/ldc/tools/ChocolateyInstall.ps1

```diff
@@ -17,3 +17,14 @@ $packageArgs = @{
 }
 
 Install-ChocolateyZipPackage @packageArgs
+
+$Bin = Join-Path $destDir "ldc2-1.24.0-windows-x86/bin"
+$Bin64 = Join-Path $destDir "ldc2-1.24.0-windows-x64/bin"

```

Comment:
this path is version dependant. Not sure how to make this compatible with automatic updating.

Reply:
Automatic updating is done in the `update.ps1` script that lives next to the `ChocolateyInstall.ps1` file. It does simple regex substitution. You can create a `$Version` variable, like we already have in `ChocolateyBeforeModify.ps1`, and add the corresponding rule to `update.ps1`.

Reply:
Fixed in https://github.com/Manuzor/chocolatey-packages/pull/1/commits/150fca1f22b9c2e578fe8d6bd693016d420a0ea9